auth/webRoutes.py

```diff
@@ -25,6 +25,32 @@
 from database.flaskAlchemyInit import log
 from controller.KafkaPublisher import send_notification
 
+from alarmlibrary.connection import RabbitMqClientConnection
+from alarmlibrary.alarm import Alarm, AlarmSeverity
+
+HTTP2ALARM = {
```

Comment:
This is only used in the context of `publish_alarm` right?
Why not have it defined within it?